### PR TITLE
Icon external circle only, excluding complete icon

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Task
Replace primary color with `#EDEEEF` for icon circles while the new icons are being implemented.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-581-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-581-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://581-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-581-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://581-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-581-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-581-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://581-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-581-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/crossDevice/Intro/assets/take-selfie.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg width="48" height="48" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><circle id="a" cx="8" cy="8" r="7.5"/></defs><g fill="none" fill-rule="evenodd"><circle fill="#EDEEEF" fill-rule="nonzero" cx="24" cy="24" r="24"/><rect stroke="#2C3E4F" stroke-width="1.333" fill="#FFF" fill-rule="nonzero" x="13" y="9" width="22" height="38" rx="2.667"/><rect fill="#2C3E4F" fill-rule="nonzero" x="20" y="11" width="8" height="1" rx=".5"/><g transform="translate(16 18)"><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><circle stroke="#2C3E4F" mask="url(#b)" cx="12.85" cy="-.85" r="7.35"/><circle stroke="#2C3E4F" mask="url(#b)" cx="-1.65" cy=".85" r="7.35"/></g><circle stroke="#2C3E4F" fill="#FFF" fill-rule="nonzero" cx="24" cy="43" r="1.5"/><circle fill="#2C3E4F" fill-rule="nonzero" cx="22" cy="27" r="1"/><circle fill="#2C3E4F" fill-rule="nonzero" cx="26" cy="27" r="1"/><circle stroke="#2C3E4F" cx="24" cy="26" r="7.5"/></g></svg>
```
Comment (josokinas) [inline src/components/crossDevice/Intro/assets/take-selfie.svg]: [Minor] Newline at the end of the file is missing.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-581-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://581-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-581-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**